src/containers/BookList.js

```diff
@@ -1,24 +1,32 @@
+/* eslint-disable consistent-return */
 /* eslint-disable react/forbid-prop-types */
```

Comment:
- [ ] Disabling `eslint` rule here is basically the same thing to change eslintrc config file. Let's not change eslint config file